teachopencadd/talktorials/T015_protein_ligand_docking/talktorial.ipynb

```diff
@@ -4,231 +4,206 @@
    "cell_type": "markdown",
```

Comment:
<p>Drop a couple comments saying how you are calculating the ligand bounding box center. Some people using the center of mass, but you are using a different one and it would be nice to clarify why and how.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>